fastcov.py

```diff
@@ -151,35 +151,55 @@ def processGcovs(cwd, gcov_files, gcov_filter_options):
     return files
 
 def dumpBranchCoverageToLcovInfo(f, branches):
-    branch_miss  = 0
+    branch_miss = 0
+    branch_found = 0
+    brda = []
     for line_num, branch_counts in branches.items():
         for i, count in enumerate(branch_counts):
-            #Branch (<line number>, <block number>, <branch number>, <taken>)
-            f.write("BRDA:{},{},{},{}\n".format(line_num, int(i/2), i, count))
+            # Branch (<line number>, <block number>, <branch number>, <taken>)
+            brda.append((int(line_num), int(i/2), i, int(count)))  # XXX: just use int for line_num/count throughout
             branch_miss += int(count == 0)
-    f.write("BRF:{}\n".format(len(branches)))                 #Branches Found
-    f.write("BRH:{}\n".format((len(branches) - branch_miss))) #Branches Hit
+            branch_found += 1
```

Comment:
Ah, fastcov wasn't counting that multiple branches can exist per line, that's how it BRH could be negative